<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VobcAlarmReport</class>
 <widget class="QWidget" name="VobcAlarmReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>设备</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="deviceComboBox">
          <item>
           <property name="text">
            <string>所有设备</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>VOBC车头/车尾</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>车载AP/LTE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>交换机</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>车辆网络</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>车辆IO</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>车载电源</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DMI</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>BTM</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>多普勒雷达</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>应答器</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>脉冲</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>设备类型</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="typeComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>不限</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>板级</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>外围</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>其他</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>故障类型</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="faultTypeComboBox">
          <item>
           <property name="text">
            <string>不限</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>硬件故障</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>软件故障</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>报警级别</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="faultLevelComboBox">
          <item>
           <property name="text">
            <string>不限</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>一级</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>二级</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>三级</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="dateTimeEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>到</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="dateTimeEdit_2"/>
        </item>
        <item>
         <widget class="QPushButton" name="findButton">
          <property name="text">
           <string>查找</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>导出</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>label</zorder>
       <zorder>typeComboBox</zorder>
       <zorder>label_2</zorder>
       <zorder>faultTypeComboBox</zorder>
       <zorder>label_3</zorder>
       <zorder>faultLevelComboBox</zorder>
       <zorder>findButton</zorder>
       <zorder>pushButton_2</zorder>
       <zorder>label_6</zorder>
       <zorder>deviceComboBox</zorder>
       <zorder>label_4</zorder>
       <zorder>dateTimeEdit</zorder>
       <zorder>label_5</zorder>
       <zorder>dateTimeEdit_2</zorder>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,2,1,1,2">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>60</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>60</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="firstButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>首页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="perviousButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>上一页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pageLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>50</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>下一页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lastButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>尾页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
